added auto generate for Magnitude

CCoupse opened the pull request.

CCoupse committed:
added auto generate for Magnitude

zainalsaputra reviewed (changes requested):
depth value masih default value, gunakan opsi random value atau dummy data
![Screenshot 2025-06-09 221809](https://github.com/user-attachments/assets/90811224-c75c-4f39-bd66-dcb3f0cdac24)

CCoupse committed:
Update app.py

Aman: Kebanyakan gempa dengan kedalaman 300–700 km, atau gempa dangkal dengan magnitude sangat rendah (<4.0).
Waspada: Gempa dengan kedalaman 70–300 km, atau gempa dangkal dengan magnitude sedang (4.0–6.0).
Berbahaya: Gempa dangkal (0–70 km) dengan magnitude tinggi (>6.0), terutama di area rawan tsunami atau struktur bangunan lemah.

zainalsaputra reviewed (approved):
great job
